gitlab/create_gitlab_webhook.sh

```diff
@@ -51,7 +51,7 @@ _create_gitlab_webhook() {
 
   if ! passw cbi "${pw_store_path}/webhook-secret" &> /dev/null ; then
     echo "Creating webhook secret credentials in password store..."
-    pwgen -1 -s -r '&\!|%{' -y 24 | passw cbi insert --echo "${pw_store_path}/webhook-secret"
+    pwgen -1 -s -r '&\!|%{'\''$' -y 24 | passw cbi insert --echo "${pw_store_path}/webhook-secret"
```

Comment:
Isn't that the same as `pwgen -s -y 24 -r '&\!|%{$'\' [...]` with two characters less?